infra/discovery/src/lib/search.js

```diff
@@ -125,6 +177,13 @@ class Listing {
       term: { valid: false }
     })
 
+    // Never return any listings moderated as hidden
+    esQuery.bool.must_not.push({
+      terms: {
+        scoreTags: ['Hide', 'hide', 'Super Hide', 'super hide']
```

Comment:
why do we need to handle different casing here ? under which scenario would the tag get lower cased ?

Reply:
Previously I tested this, and the lower case version seemed to be required. I've tested now and that is at least now no longer the case.  I've removed the lowercase names.